.github/labeler.yml

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+build:
+  - changed-files:
+    - any-glob-to-any-file:
+      - '.github/**'
+      - 'gradle/**'
+      - '*gradlew*'
+
+clients:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'clients/**'
+
+connect:
+- changed-files:
+  - any-glob-to-any-file: 
+    - 'connect/**'
```

Comment:
Can we use the separate [mirror-maker-2 label](https://github.com/apache/kafka/pulls?q=is%3Apr+label%3Amirror-maker-2) for changes to `connect/mirror`?

Reply:
Added. But in this case, the labeler still attach connect to connect/mirror.

Reply:
Yeah, it'd be nice if we could avoid that. MM2 and Connect are about as different as Connect and the clients module.

Maybe we can change the `connect` label to match any of the Connect submodules except for `connect/mirror/**`?

Reply:
Unfortunally, the tool did not so complex thing.
I have tried follow code but it does not work.
```
connect:
- changed-files:
  - all-globs-to-any-file: 
    - 'connect/**'
    - 'connect/[!mirror]*/**'
```

Reply:
I think the [docs](https://github.com/actions/labeler) give an example that would work for this case:

```
# Add 'source' label to any change to src files within the source dir EXCEPT for the docs sub-folder
source:
- all:
  - changed-files:
    - any-glob-to-any-file: 'src/**/*'
    - all-globs-to-all-files: '!src/docs/*'
```

Reply:
If nothing else works, we can just manually specify one glob for every submodule that isn't MM2; e.g., `connect/runtime/**`, `connect/api/**`, `connect/json/**`, etc.

Reply:
> - any-glob-to-any-file: 'src/**/*'
>     - all-globs-to-all-files: '!src/docs/*'

Thanks, I missed this one.  Updated.